src/components/shared/title/title.component.jsx

```diff
@@ -35,7 +35,7 @@ function Title ({
       )
     case 'h4':
       return (
-        <h4 className={classes.title}>
+        <h4 className={`${classes.title} ${classes.titleSecondary}`}>
```

Comment:
An `h4` heading will always be a `secondary` title? Or can we have an `h2` secondary title for example?

Reply:
Chami did not specify, so I added some headings for semantics and a11y. I would leave it like this and if we have any other needs (which I doubt) we can refactor this.